doc/user/events/widget.rst

```diff
@@ -334,19 +334,21 @@ Hosted or pretix Enterprise are active, you can pass the following fields:
 * If you use the campaigns plugin, you can pass a campaign ID as a value to ``data-campaign``. This way, all orders
   made through this widget will be counted towards this campaign.
 
-* If you use the tracking plugin, you can enable cross-domain tracking. Use the HTML code to embed the widget and 
-  add one the following code snippets. Make sure to replace all occurrences of <MEASUREMENT_ID> with your Google 
-  Analytics MEASUREMENT_ID (G-XXXXXXXX)
+* If you use the tracking plugin, you can enable cross-domain tracking. Please note: when you run your pretix-shop on a
+  subdomain of your main tracking domain, then you do not need cross-domain tracking as tracking automatically works
+  across subdomains. See `Using a custom domain <https://docs.pretix.eu/en/latest/user/organizers/domain.html>` for
```

Comment:
That should be a sphinx crossref, not a fully qualified link:
https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html

Reply:
Thanks! Should be fixed. :)